describedby-submenu.html

```diff
@@ -0,0 +1,170 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Navigation Example</title>
+  <link rel="stylesheet" href="node_modules/normalize.css/normalize.css">
+  <link rel="stylesheet" href="css/base.css">
+  <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+</head>
+<body class="nav-list-example">
+
+  <header role="banner">
+    <h1>Navbar with aria-describedby submenus</h1>
+  </header>
+
+  <div class="flexContainer flexSpaceAround">
+      <div class="col">
+        <nav class="pf-c-vertical-nav">
+            <ul class="pf-c-vertical-nav__content" role="list">
+              <li class="pf-c-vertical-nav__item">
+
+                <a href="#" class="pf-c-vertical-nav__link">
+                  <span class="pf-c-vertical-nav__link-icon"><i class="fas fa-home"></i></span>
+                  <span class="pf-c-vertical-nav__link-text">Home</span>
+                </a>
+              </li>
+
+              <li class="pf-c-vertical-nav__item">
+                <a href="#" class="pf-c-vertical-nav__link">
+                  <span class="pf-c-vertical-nav__link-icon"><i class="fas fa-home"></i></span>
+                  <span class="pf-c-vertical-nav__link-text">Technology</span>
+                </a>
+              </li>
+
+              <li class="pf-c-vertical-nav__item">
+
+                <a
+                  href="#"
+                  aria-expanded="false"
+                  aria-controls="navbarSubmenu3"
+                  aria-current="false"
+                  class="pf-c-vertical-nav__link"
+                  id="navbarDropdownMenuLink3">
+                  <span class="pf-c-vertical-nav__link-icon">
+                    <i class="fas fa-home"></i>
+                  </span>
+                  <span class="pf-c-vertical-nav__link-text">
+                    Science Menu Item
+                  </span>
+                </a>
+
+                <section
+                  class="pf-vertical-sub-nav"
+                  hidden
+                  aria-label="Science"
+                  id="navbarSubmenu3">
+                  <h2 class="pf-vertical-sub-nav__title">Science Heading</h2>
+                  <ul class="pf-vertical-sub-nav__content" aria-describedby="navbarSubmenu3">
```

Comment:
I'm not seeing aria-describedby provide any information for the list when I test in VO. I came across this article: https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

According to their research, there are certain roles for which these attributes are effective, and since `<ul>` is a `list` role, which is [not an interactive role](https://www.w3.org/TR/wai-aria-1.1/#document_structure_roles), I don't think this is a reliable method to use.

Also, in my own testing (VO in Safari), I found that the way `<ul>` and `<li>` elements are announced in VO can be affected by factors like how many other elements are included in the `<li>`. I found that I could "fix" some of the behavior by adding a redundant `role` attribute for these elements. It's almost like including too many miscellaneous elements makes it difficult for VO to understand what these elements are.

I also am not sure that setting aria-described by to an id where the text is defined in aria-label would work reliably. This would be an interesting thing to test in an isolated test case for aria-label, aria-labelledby and aria-describedby attributes that we had discussed.

Reply:
Great feedback, and thanks for sharing those links - super helpful. I was seeing some success with aria-describedby in this case, but not in Safari now that I think about it.

I'll move this example to an isolated test page that tests `aria-describedby` in several other contexts as you suggested.